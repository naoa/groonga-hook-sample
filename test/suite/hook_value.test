register commands/hook

table_create Entries TABLE_HASH_KEY ShortText
column_create Entries title COLUMN_SCALAR ShortText

table_create Tokens TABLE_PAT_KEY ShortText --default_tokenizer TokenBigram
column_create Tokens entries_title COLUMN_INDEX|WITH_POSITION Entries title

example_hook_add_value --table Entries --column title --value "hogehoge"
example_hook_get_value --table Entries --column title
example_hook_delete_value --table Entries --column title

