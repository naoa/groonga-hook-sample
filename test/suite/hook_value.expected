register commands/hook
[[0,0.0,0.0],true]
table_create Entries TABLE_HASH_KEY ShortText
[[0,0.0,0.0],true]
column_create Entries title COLUMN_SCALAR ShortText
[[0,0.0,0.0],true]
table_create Tokens TABLE_PAT_KEY ShortText --default_tokenizer TokenBigram
[[0,0.0,0.0],true]
column_create Tokens entries_title COLUMN_INDEX|WITH_POSITION Entries title
[[0,0.0,0.0],true]
example_hook_add_value --table Entries --column title --value "hogehoge"
[[0,0.0,0.0],[1]]
example_hook_get_value --table Entries --column title
[[0,0.0,0.0],["hogehoge"]]
example_hook_delete_value --table Entries --column title
[[0,0.0,0.0],[0]]
